templates/request/get/get.handlebars

```diff
@@ -72,3 +72,67 @@
         done();
       });
     });
+{{#if isLoadTest}}
+    it('load tests with {{length description}}', function(done) {
+      arete.loadTest({
+        name: 'PUT THE NAME HERE',
+        requests: {{requests}},
+        concurrentRequests: {{concurrent}},
+        targetFunction: function(callback) {
+          request({
+            url: '{{pathify path}}',
+            {{#ifCond queryParameters queryApiKey}}
+            qs: {
+              {{#if queryApiKey}}{{queryApiKey.type}}: process.env.{{queryApiKey.name}}{{#if queryParameters}},
+              {{/if}}{{/if}}{{#if queryParameters}}{{#each queryParameters}}{{this.name}}: 'DATA GOES HERE'{{#unless @last}},{{/unless}}{{/each}}{{/if}}
+            },
+            {{/ifCond}}
+            method: 'GET',
+            headers: {
+              'Content-Type': '{{contentType}}'{{#if headerParameters}},
+              'Custom-Header': { {{#each headerParameters}}
```

Comment:
What is  'Custom-Header'? Doesn't each `headerParameters` has a name attribute than can be used for header names here?